examples/component/http-server/README.md

```diff
@@ -0,0 +1,179 @@
+# TinyGo HTTP Password Checker
+
+This repository contains a WebAssembly Component written in [TinyGo][tinygo], which:
+
+- Implements a [`wasi:http`][wasi-http]-compliant HTTP handler
+- Uses the [`httpserver` provider][httpserver-provider] to serve requests
+- Can be declaratively provisioned with [`wadm`][wadm]
+
+[wasi-http]: https://github.com/WebAssembly/wasi-http
+[httpserver-provider]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-http-server
+[wadm]: https://github.com/wasmCloud/wadm
+[tinygo]: https://tinygo.org/getting-started/install/
+[wash]:  https://wasmcloud.com/docs/ecosystem/wash/
+
+## Dependencies
+
+Before starting, ensure that you have the following installed:
+
+- The [TinyGo toolchain][tinygo]
+- [`wash`, the WAsmcloud SHell][wash] installed.
+
+## Quickstart
+
+To get started developing this repository quickly, clone the repo and run `wash dev`:
+
+```console
+wash dev
+```
+
+`wash dev` does many things for you:
+
+- Starts the [wasmCloud host][wasmcloud-host] that can run your WebAssembly component
+- Builds this project
+- Builds a declarative WADM manifest consisting of:
+  - Your locally built component
+  - A [HTTP server provider][httpserver-provider] which will receive requests from the outside world
+    (on port 8000 by default)
+  - Necessary links between providers and your component so your component can handle web traffic
+- Deploys the built manifest (i.e all dependencies to run this application) locally
+- Watches your code for changes and re-deploys when necessary.
+
+[wasmcloud-host]: https://wasmcloud.com/docs/concepts/hosts
+
+## Send a request to the running component
+
+Once `wash dev` is serving your component, to send a request to the running component (via the HTTP
+server provider)
+
+### Available Endpoints
+
+#### GET /
+
+Returns a list of available endpoints and their descriptions.
+
+```console
+curl http://localhost:8000/
+  /error - return a 500 error
+  /form - echo the fields of a POST request
+  /headers - echo your user agent back as a server side header
+  /post - echo the body of a POST request
+```
```

Comment:
This example uses [`30000` for the port](https://github.com/wasmCloud/go/blob/main/examples/component/http-server/wadm.yaml#L37), would you mind either updating the `README` or `wadm.yaml` to match.

Reply:
Same here around wash dev using port 8000. Can change wadm.yaml